apps/backoffice/mocks/handlers/selfcare-handlers.ts

```diff
@@ -29,7 +29,16 @@ export const buildHandlers = () => {
     }),
     rest.get(`${baseURL}/institutions`, (_, res, ctx) => {
       const resultArray = [
-        [ctx.status(200), ctx.json([getMockInstitution()])],
+        [
+          ctx.status(200),
+          ctx.json(
+            [
+              ...Array.from(
+                Array(faker.number.int({ min: 140, max: 145 })).keys()
```

Comment:
**question:** are we sure we want `msw` to return all this authorized institutions?!

Reply:
leftover msw mock from a test, removed